### ISSUE TITLE
Document Events for Login/Logout

### ISSUE THREAD

**Issue by jensens** [plone/documentation#871]
they are missing in https://5.docs.plone.org/develop/addons/components/events.html#event-types
see also https://stackoverflow.com/questions/43852342/is-there-some-logout-hook-in-plone-4/43853802#43853802

**Comment by tobiasherp** [comment]
There should be a sentence about handling event interfaces vs. event classes as well.

**Comment by jensens** [comment]
Indeed, the example here https://5.docs.plone.org/develop/addons/components/events.html#subscribing-using-zcml uses the event class. This works, but best practice is to use the event interface.
